RELEASES.md

```diff
@@ -28,9 +28,9 @@ version="[1.1,1.2)"
 
 ## Serviceability
 
-When we release a new MINOR version, the previous MINOR version is abandoned.
+When we release a new MINOR version, the previous MINOR versions shipped within one year MAY still accept bug report and receive patches. MINOR versions more than one year old will not receive any support. If you suspect there is an issue in a 1+ year old version that you are using, please upgrade to latest MINOR version and retry, before you send out a bug report.
 
-When we release a new MAJOR version, we will continue to apply bug fixes to the existing features in the previous MAJOR version for up to the 2 year support cycle for Azure.
+When we release a new MAJOR version, we will continue to apply bug fixes to the existing features in the previous MAJOR version for up to the 1 year support cycle for Azure.
 Example: We release MSALiOS 2.0 in the future which supports unified Auth for AAD and MSA. Later, we then have a fix in Conditional Access for MSALiOS. Since that feature exists both in MSALiOS 1.1 and MSALiOS 2.0, we will fix both. It will roll up in a PATCH number for each. Customers that are still locked down on MSALiOS 1.1 will receive the benefit of this fix.
 
 ## Microsoft Authentication SDKs and Azure Active Directory
```

Comment:
A couple lines down it also says "MAJOR version will continue to work for 2 years after release", and in the table with descriptions/examples above it mentions "We will continue to service MAJOR version numbers with bug fixes up to the two year support cycle"

Both of those sections are also talking about Azure AD support, so is the 2 years of support there different from MSAL's 1 year support?

Reply:
Good catch. I modified it with a new commit now.